pkg/apis/crd/v1alpha1/types.go

```diff
@@ -618,3 +627,8 @@ type ICMPProtocol struct {
 	ICMPType *int32 `json:"icmpType,omitempty"`
 	ICMPCode *int32 `json:"icmpCode,omitempty"`
 }
+
+type IGMPProtocol struct {
+	IGMPType     *int32 `json:"igmpType,omitempty"`
```

Comment:
If nil means all IGMP types, please add a comment to explain that.

Reply:
IGMPType can only be IGMP query or IGMP report

Reply:
If IGMPType cannot be nil, why use `*int32`? It should be int32 and not optional.